src/core/default/resources/simple/xlambda.py

```diff
@@ -366,7 +372,7 @@ def __call__(self, *args: Any, **kwds: Any) -> Any:
 
 def simple_function_annotation(
     name: str,
-    events: List[Event] = [],
+    events: List[Union[Event, RouteEvent]] = [],
```

Comment:
`RouteEvent` should derive from `Event`, but it does not right now. Ill look into if there was a reason for that or if it is just an oversight.

Reply:
I thought so. But didn't have time to dig into it yet